<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>999</width>
    <height>650</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>WeatherStation</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <layout class="QGridLayout" name="gridLayout_4">
      <property name="sizeConstraint">
       <enum>QLayout::SetMaximumSize</enum>
      </property>
      <item row="0" column="2">
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="2" column="0">
         <widget class="QGroupBox" name="groupBox_2">
          <property name="title">
           <string>Fréquences d'échantillonage</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_5">
           <property name="sizeConstraint">
            <enum>QLayout::SetDefaultConstraint</enum>
           </property>
           <property name="topMargin">
            <number>3</number>
           </property>
           <property name="bottomMargin">
            <number>3</number>
           </property>
           <property name="verticalSpacing">
            <number>3</number>
           </property>
           <item row="0" column="0">
            <widget class="QLabel" name="label">
             <property name="text">
              <string>Fréquence Détecteur 1 (Hz)</string>
             </property>
            </widget>
           </item>
           <item row="3" column="0">
            <widget class="QLabel" name="label_4">
             <property name="text">
              <string>Fréquence mode 2 (s)</string>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="label_3">
             <property name="text">
              <string>Fréquence Détecteur 3 (Hz)</string>
             </property>
            </widget>
           </item>
           <item row="1" column="2">
            <widget class="QSlider" name="sensor2Slider">
             <property name="minimum">
              <number>1</number>
             </property>
             <property name="maximum">
              <number>20</number>
             </property>
             <property name="pageStep">
              <number>10</number>
             </property>
             <property name="value">
              <number>1</number>
             </property>
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="label_2">
             <property name="text">
              <string>Fréquence Détecteur 2 (Hz)</string>
             </property>
            </widget>
           </item>
           <item row="0" column="3">
            <widget class="QLCDNumber" name="lcdNumber">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="frameShape">
              <enum>QFrame::NoFrame</enum>
             </property>
             <property name="frameShadow">
              <enum>QFrame::Plain</enum>
             </property>
             <property name="smallDecimalPoint">
              <bool>true</bool>
             </property>
             <property name="digitCount">
              <number>2</number>
             </property>
             <property name="segmentStyle">
              <enum>QLCDNumber::Flat</enum>
             </property>
             <property name="intValue" stdset="0">
              <number>1</number>
             </property>
            </widget>
           </item>
           <item row="1" column="3">
            <widget class="QLCDNumber" name="lcdNumber_2">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="frameShape">
              <enum>QFrame::NoFrame</enum>
             </property>
             <property name="frameShadow">
              <enum>QFrame::Plain</enum>
             </property>
             <property name="smallDecimalPoint">
              <bool>true</bool>
             </property>
             <property name="digitCount">
              <number>2</number>
             </property>
             <property name="segmentStyle">
              <enum>QLCDNumber::Flat</enum>
             </property>
             <property name="intValue" stdset="0">
              <number>1</number>
             </property>
            </widget>
           </item>
           <item row="2" column="2">
            <widget class="QSlider" name="sensor3Slider">
             <property name="minimum">
              <number>1</number>
             </property>
             <property name="maximum">
              <number>20</number>
             </property>
             <property name="value">
              <number>1</number>
             </property>
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
            </widget>
           </item>
           <item row="4" column="0">
            <widget class="QPushButton" name="setFrequenciesButton">
             <property name="text">
              <string>Set frequencies</string>
             </property>
            </widget>
           </item>
           <item row="0" column="2">
            <widget class="QSlider" name="sensor1Slider">
             <property name="minimum">
              <number>1</number>
             </property>
             <property name="maximum">
              <number>20</number>
             </property>
             <property name="value">
              <number>1</number>
             </property>
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
            </widget>
           </item>
           <item row="2" column="3">
            <widget class="QLCDNumber" name="lcdNumber_3">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="frameShape">
              <enum>QFrame::NoFrame</enum>
             </property>
             <property name="frameShadow">
              <enum>QFrame::Plain</enum>
             </property>
             <property name="smallDecimalPoint">
              <bool>true</bool>
             </property>
             <property name="digitCount">
              <number>2</number>
             </property>
             <property name="segmentStyle">
              <enum>QLCDNumber::Flat</enum>
             </property>
             <property name="intValue" stdset="0">
              <number>1</number>
             </property>
            </widget>
           </item>
           <item row="3" column="2">
            <widget class="QSpinBox" name="mode2Frequency">
             <property name="suffix">
              <string/>
             </property>
             <property name="prefix">
              <string/>
             </property>
             <property name="minimum">
              <number>1</number>
             </property>
             <property name="maximum">
              <number>5</number>
             </property>
             <property name="singleStep">
              <number>1</number>
             </property>
             <property name="value">
              <number>1</number>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QGroupBox" name="groupBoxMode">
          <property name="title">
           <string>Selection du mode</string>
          </property>
          <property name="flat">
           <bool>false</bool>
          </property>
          <property name="checkable">
           <bool>false</bool>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout">
           <property name="spacing">
            <number>3</number>
           </property>
           <property name="topMargin">
            <number>3</number>
           </property>
           <property name="bottomMargin">
            <number>3</number>
           </property>
           <item>
            <widget class="QRadioButton" name="mode0Button">
             <property name="text">
              <string>Arrêt</string>
             </property>
             <property name="checked">
              <bool>true</bool>
             </property>
             <attribute name="buttonGroup">
              <string notr="true">modeGroup</string>
             </attribute>
            </widget>
           </item>
           <item>
            <widget class="QRadioButton" name="mode1Button">
             <property name="text">
              <string>Mode 1</string>
             </property>
             <property name="checkable">
              <bool>true</bool>
             </property>
             <property name="checked">
              <bool>false</bool>
             </property>
             <property name="autoRepeat">
              <bool>false</bool>
             </property>
             <attribute name="buttonGroup">
              <string notr="true">modeGroup</string>
             </attribute>
            </widget>
           </item>
           <item>
            <widget class="QRadioButton" name="mode2Button">
             <property name="text">
              <string>Mode 2</string>
             </property>
             <attribute name="buttonGroup">
              <string notr="true">modeGroup</string>
             </attribute>
            </widget>
           </item>
           <item>
            <widget class="QRadioButton" name="mode3Button">
             <property name="text">
              <string>Mode 3</string>
             </property>
             <attribute name="buttonGroup">
              <string notr="true">modeGroup</string>
             </attribute>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="setModeButton">
             <property name="text">
              <string>Set Mode</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="mode3DataPushButton">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="text">
              <string>Get mode 3 data</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item row="1" column="0">
     <widget class="QtCharts::QChartView" name="chartView">
      <property name="renderHints">
       <set>QPainter::Antialiasing|QPainter::TextAntialiasing</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>999</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFichier">
    <property name="title">
     <string>Station</string>
    </property>
    <addaction name="actionQuitter"/>
   </widget>
   <widget class="QMenu" name="menuA_propos">
    <property name="title">
     <string>A propos</string>
    </property>
    <addaction name="actionA_propos"/>
    <addaction name="actionA_propos_de_Qt"/>
   </widget>
   <addaction name="menuFichier"/>
   <addaction name="menuA_propos"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionQuitter">
   <property name="text">
    <string>Quitter</string>
   </property>
  </action>
  <action name="actionA_propos_de_Qt">
   <property name="text">
    <string>A propos de Qt</string>
   </property>
  </action>
  <action name="actionA_propos">
   <property name="text">
    <string>A propos</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QtCharts::QChartView</class>
   <extends>QGraphicsView</extends>
   <header location="global">QChartView</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>sensor3Slider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>lcdNumber_3</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>678</x>
     <y>129</y>
    </hint>
    <hint type="destinationlabel">
     <x>702</x>
     <y>131</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>sensor2Slider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>lcdNumber_2</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>678</x>
     <y>103</y>
    </hint>
    <hint type="destinationlabel">
     <x>712</x>
     <y>107</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>setModeButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>setMode()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>873</x>
     <y>173</y>
    </hint>
    <hint type="destinationlabel">
     <x>894</x>
     <y>152</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>sensor1Slider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>lcdNumber</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>489</x>
     <y>72</y>
    </hint>
    <hint type="destinationlabel">
     <x>704</x>
     <y>68</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>setFrequenciesButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>sendFrequencies()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>166</x>
     <y>188</y>
    </hint>
    <hint type="destinationlabel">
     <x>895</x>
     <y>212</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>mode3DataPushButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>getMode3Data()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>833</x>
     <y>186</y>
    </hint>
    <hint type="destinationlabel">
     <x>894</x>
     <y>191</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionQuitter</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>448</x>
     <y>256</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionA_propos</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>about()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>448</x>
     <y>256</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionA_propos_de_Qt</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>aboutQt()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>448</x>
     <y>256</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>setMode()</slot>
  <slot>sendFrequencies()</slot>
  <slot>getMode3Data()</slot>
  <slot>aboutQt()</slot>
  <slot>about()</slot>
 </slots>
 <buttongroups>
  <buttongroup name="modeGroup"/>
 </buttongroups>
</ui>
